### ISSUE TITLE
Don't cross-post Retweets

### ISSUE THREAD

**Issue by FlowFX** [FlowFX/twit2toot#15]
Retweets should stay where they are, on Twitter. They would only pollute Mastodon timelines.

**Comment by FlowFX** [comment]
@Cheatha Stimmst du dem zu? Oder denke ich da falsch?

**Comment by Cheatha** [comment]
Ich bin unschlüssig: Wenn man die Toots mit »example@twitter.com schreibt:\n« beginnt, die URLs umschreibt und Bilder neu hochlädt, könnte es ok sein. Vielleicht einfach mal zurückstellen und kucken, welche Erfahrungen die normale Bridge bringt?

**Comment by FlowFX** [comment]
Ja. Erstmal gucken. Mich nerven die RT-Toots von anderen Services schon heute.
